nextflow.config

```diff
@@ -23,6 +23,7 @@ params {
     hificnv_xx                 = null
     hificnv_exclude            = null
     somalier_sites             = null
+    deepvariant_model_type     = params.preset == 'ONT_R10' ? 'ONT_R104' : 'PACBIO'
```

Comment:
Won't this always overwrite the `deepvariant_model_type` variable

Reply:
I don't think so, if I run `--preset ONT_R10 --deepvariant_model_type WES` I end up with WES as the model type in DeepVariant.